### DISCUSSION TITLE
Error running npx prisma init

### DISCUSSION THREAD

**Discussion by AllThingsSmitty** [prisma/prisma#25093] [Q&A]
### Question

I'm running `npx prisma init` and receiving the following error in the terminal:

```
0%Error: request to https://binaries.prisma.sh/all_commits/78caf6feeaed953168c64e15a249c3e9a033ebe2/windows/query_engine.dll.node.gz.sha256 failed, reason: read ECONNRESET
```

Following a [past discussion](https://github.com/prisma/prisma/discussions/19915) I set my `.env` and `.env.local` files to incluide the following:

```
PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1
```

However, I'm still receiving the same error:

```
0%Error: request to https://binaries.prisma.sh/all_commits/78caf6feeaed953168c64e15a249c3e9a033ebe2/windows/query_engine.dll.node.gz.sha256 failed, reason: read ECONNRESET
```

### How to reproduce (optional)

Run `npx prisma init`

### Expected behavior (optional)

Prisma will create a local database instance for the project.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}```

- OS: Windows
- Database: SQLite 
- Node.js version: 21.6.2

**Comment by ludralph** [comment]
Hi @AllThingsSmitty 👋 

If you click on the  Prisma binary link  in your browser, does the engine get downloaded?
```
https://binaries.prisma.sh/all_commits/78caf6feeaed953168c64e15a249c3e9a033ebe2/windows/query_engine.dll.node.gz.sha256
```
Are you behind a firewall or VPN?

**Reply by AllThingsSmitty** [reply]
@ludralph yes I can download the file.

I'm on my company firewall.

**Reply by ludralph** [reply]
Can you try with another network to see if the issue persist?

**Reply by AllThingsSmitty** [reply]
Sure, can do.

I was hoping that since I could download the physical file it would be something other than a network proxy matter.

**Reply by ludralph** [reply]
This [comment](https://github.com/prisma/prisma/discussions/18873#discussioncomment-5927340) explains why you can't download it from the CLI

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
